symforce/codegen/backends/README.md

```diff
@@ -0,0 +1,14 @@
+# Code Generation Backends
+
+SymForce takes symbolic functions and generates runtime functions for multiple target backends. It aims to make it very straightforward to add new backends.
+
+The minimal steps to support a new backend are:
```

Comment:
add links to sympy printing https://docs.sympy.org/latest/modules/printing.html and jinja https://realpython.com/primer-on-jinja-templating/